### PR TITLE
Add tests for "disable_ac" feature

### PR DESCRIPTION
**Pull request by jcalderin**

### CODE REVIEW

File: test/acceptance/integration/steps_lib/background.py [hunk 1]
```diff
@@ -179,6 +181,32 @@ def the_bypass_configuration(step):
...
         time.sleep(5)
 
+@step("the Context Broker configuration with Access Control and check headers functionality disabled")
+def the_context_broker_configuration_with_Access_control_adn_check_headers_functionality_disabled(step):
+    """
+    Set the context broker configuration and disable the checks with AC and the checks of the headers
+    :param step:
+    :return:
+    """
+    if world.config_set != 'cb_without_ac_and_headers':
```
Comment (iariasleon) [inline test/acceptance/integration/steps_lib/background.py]: Is it necessary this condition?

File: test/acceptance/integration/steps_lib/background.py [hunk 2]
```diff
@@ -179,6 +181,32 @@ def the_bypass_configuration(step):
...
+        time.sleep(5)
+
+@step("the Context Broker configuration with Access Control disabled$")
+def the_context_broker_configuration_with_Access_control_adn_check_headers_functionality_disabled(step):
+    """
+    Set the context broker configuration and disable the checks with AC and the checks of the headers
+    :param step:
+    :return:
+    """
+    if world.config_set != 'cb_without_ac':
```
Comment (iariasleon) [inline test/acceptance/integration/steps_lib/background.py]: Is it necessary this condition?

**Comment by fgalan** [comment]
LGTM (passing the ball to @dmoranj for final LGTM + merge + delete branch)

**Comment by iariasleon** [comment]
LGTM

**Pull request merged**